exercises/poker/package.yaml

```diff
@@ -0,0 +1,20 @@
+name: poker
+version: 1.1.1.0
```

Comment:
https://github.com/exercism/problem-specifications/blob/master/exercises/poker/canonical-data.json says 1.1.0 so thi should be 1.1.0.1

Reply:
I placed the 1 on the other side, that was dumb